hash/snes.xml

```diff
@@ -44636,7 +44636,7 @@ List of unclassified roms
 		</part>
```

Comment:
Please remove this line, it's a violation of the XML standard.

Reply:
@MoochMcGee Please stop reviewing other peoples' pull requests. You are not a MAME developer, you are an external contributor. Reviewing pull requests is the purview of the core team only.

Reply:
If that's really the case then I do believe that you can disable commentary by role in github settings (is it?)

Reply:
No, anyone should be allowed to comment. External contributors need to be careful with their wording to not come across as they have the right to make demands. Especially @MoochMcGee who has been banned before for making rude comments on a PR.

On-topic: I see nothing wrong with snes.xml line 44636

Reply:
I've scanned the new files with xmllint - no format errors are showing up in that or in my check for TAB/Tag formatting.

Reply:
The problem is that GitHub is displaying it strangely if you expand the diff in the browser – it makes it look like there’s a stray `</part>` when there isn’t.  GitHub still gets weird when viewing large files.